docs/data/base/components/button/UnstyledButtonIntroduction/css/index.js

```diff
@@ -54,42 +56,62 @@ function Styles() {
 
   return (
     <style>{`
-  .IntroductionButton {
-    font-family: 'IBM Plex Sans', sans-serif;
-    font-weight: 600;
-    font-size: 0.875rem;
-    line-height: 1.5;
-    background-color: ${cyan[500]};
-    padding: 8px 16px;
-    border-radius: 8px;
-    color: white;
-    transition: all 150ms ease;
-    cursor: pointer;
-    border: 1px solid ${cyan[500]};
-    box-shadow: 0 2px 1px ${
-      isDarkMode ? 'rgba(0, 0, 0, 0.5)' : 'rgba(45, 45, 60, 0.2)'
+      .Button {
+        all: unset;
+        align-items: center;
+        background-clip: padding-box;
+        background-color: ${cyan[500]};
+        border-color: ${cyan[500]};
+        border-radius: .5rem;
+        border-style: solid;
+        border-width: .063rem;
```

Comment:
Oof, does it make sense to have this broken decimal value for the sake of `rem` consistency? 🤔

Reply:
Which one is broken?

Reply:
Was referring to the `.063`value, and "broken" not as in "not working" but rather "weird decimal value" 😅 which I know converts to 1px, but just wondering.

Reply:
Ah ok, gotcha. It does look a little weird I guess, but it's just how it works 🤷‍♂️ It's not for consistency, it's so the border-width remains fluid, like the height and border-radius. If the user zooms in, or increases/decreases their default browser font-size, we want the button to scale uniformly.